tools/tck-report-comparison/README.adoc

```diff
@@ -4,16 +4,27 @@ Tools for TCK result comparison.
 
 == Options
 
-JUnit XML and CSV formats are supported.
+JUnit XML, Cucumber JSON and CSV formats are supported.
+
+IMPORTANT: JUnit XML need to contain test display name. This was
+https://github.com/junit-team/junit5/issues/990[available before JUnit 5.1.1 version]
+and is https://issues.apache.org/jira/browse/SUREFIRE-1546[fixed in Surefire 3.0.0-M4] (not released yet).
+If version change is not possible - use link:../tck-reporting[Cucumber report].
```

Comment:
This link is dead. Otherwise the changes from this commit look good. A (working) description how to get the cucumber reports would be good though, also for me personally, since I have not done that before.

Reply:
The link is dead because Cucumber reporting [needs to be merged](https://github.com/opencypher/openCypher/pull/360) first. There is a description of how to get reporting working. Maybe you have time to review this PR?